cumulus_library/studies/core/core_templates/core_utils.jinja

```diff
@@ -0,0 +1,38 @@
+{%- macro comma_delineate(loop) -%}
+{%- if not loop.last -%}
+, 
+{%- endif -%}
+{%- endmacro -%}
+
+-- gets a list of columns from a known present source
+{% macro basic_cols(table, alias, targets) %}
+        {%- for col in targets %}
+        {{ alias }}.{{ col.lower() }}
+        {{- comma_delineate(loop) }}
+        {%- endfor %}
+{%- endmacro %}
+
+
+-- gets a column, if present, from a basic SQL primitive col, or a null otherwise
+{% macro basic_nullable_cols(table, alias, targets, schema) %}
+        {%- for col in targets %}
+        {%- if schema[table][col] %}
+        {{ alias }}.{{ col.lower() }}
```

Comment:
nit: I'm pretty sure you don't need the `.lower()` here, but it's harmless (re: us talking about case in actual SQL vs string comparison)

Reply:
i'd rather make any information_schema lookups downstream of this default to all lowercase, as that's best practice when you're not trying to maintain an external data model's semantics.